src/main/java/org/jabref/gui/preferences/external/ExternalTabViewModel.java

```diff
@@ -90,7 +90,7 @@ public ExternalTabViewModel(DialogService dialogService, PreferencesService pref
     @Override
     public void setValues() {
         eMailReferenceSubjectProperty.setValue(initialExternalApplicationPreferences.getEmailSubject());
-        autoOpenAttachedFoldersProperty.setValue(initialExternalApplicationPreferences.shouldAutoOpenEmailAttachmentsFolder());
+        autoOpenAttachedFoldersProperty.setValue(!initialExternalApplicationPreferences.shouldAutoOpenEmailAttachmentsFolder());
```

Comment:
this does not make sense. Whenever the user checks the checkbox, it will not be opened?

You just need to adjust it in the JabRefPreferences itself, change defaults value for the property

Reply:
Thank you for your feedback. I have taken it into consideration and made some changes, which I already pushed. I hope the new solution will be better.